bedrock/firefox/templates/firefox/privacy/book.html

```diff
@@ -53,7 +53,7 @@ <h1 class="mzp-c-hero-title">{{ ftl('privacy-book-little-book-of') }}</h1>
 </section>
 
 {% call split(
-  image_url='img/firefox/privacy/book/privacy-healthy-internet.png',
+  image_url='img/firefox/privacy/book/privacy-healthy-internet.jpg',
```

Comment:
Whilst there's definitely an improvement in these image sizes, the ones used by the `split` macro would still benefit from `loading="lazy"`. To do this you would need to add a new `loading` parameter to the `split` macro here: https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/macros-protocol.html#L75

I'd suggest making the default value for the parameter `eager`, and then individual pages can specify `lazy` when required, which can be applied to the `<img>` elements in the macro. Let me know if you have any questions on how to do this.

Reply:
Ok, thanks! I'll look into this and follow up if I have any questions.

Reply:
Got the remainder of the images loading lazily. Thanks for the tips!

![Screen Shot 2021-11-04 at 10 33 55 AM](https://user-images.githubusercontent.com/20748598/140360902-20acea71-ab97-4197-a013-2fff2ae7ee76.png)